packages/react-mutation-mapper/src/component/mutationMapper/MutationMapper.tsx

```diff
@@ -188,34 +188,54 @@ export default class MutationMapper<
         return this.getTrackDataStatus();
     }
 
+    protected getPlotTopYAxisSymbol(): string | undefined {
+        return this.props.plotTopYAxisSymbol;
+    }
+
     @computed
     protected get plotTopYAxisSymbol(): string | undefined {
-        return this.props.plotTopYAxisSymbol;
+        return this.getPlotTopYAxisSymbol();
```

Comment:
why do we wrap this?

Reply:
and all the rest?  just curious

Reply:
pretty sure this is the workaround for having inheritance/overriding, since we can no longer override a @computed, we have to have subclasses override the method, in this case `getPlotTopYAxisSymbol`. right @onursumer ?

Reply:
Actually @leexgh implemented this part. I think this is because of the issue with overriding the `@computed` values. mobx 6 doesn't allow that anymore. @leexgh can you confirm?

Ideally we should refactor mutation mapper so that we don't use inheritance anymore. Again, beyond the scope of this PR.

Reply:
Oh, actually, I guess these didn't need to be wrapped as `@computed` values at the first place. @alisman Did you mean that? It's probably my bad to implement this initially this way.

Reply:
@onursumer @alisman Yes this is the workaround for having overriding as @adamabeshouse said above.

Reply:
Just removed the `@computed` decoration for these functions. I don't think they need to be computed. This way we don't need to introduce the additional `get...` functions.